### PR TITLE
[VEN-2752][VEN-2751][VEN-2829]: add vip for rewards on zksync mainnet and bootstrap liquidity  of XVS

### PR DESCRIPTION
**Pull request by GitGuru7**
## Description

Resolves VEN

### CODE REVIEW

File: multisig/proposals/zksyncmainnet/vip-010/index.ts [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [
+        ethers.constants.AddressZero,
+        "setLastRewardingBlockTimestamps(address[],uint256[],uint256[])",
+        zksyncmainnet.GUARDIAN,
+      ],
+    },
+
+    // Configure pool rewards
```
Comment (chechu) [inline multisig/proposals/zksyncmainnet/vip-010/index.ts:45]:
```suggestion
    // Configure XVS vault rewards
```

File: multisig/proposals/zksyncmainnet/vip-010/index.ts [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+import { ethers } from "hardhat";
+import { NETWORK_ADDRESSES } from "src/networkAddresses";
+import { makeProposal } from "src/utils";
+
+const { zksyncmainnet } = NETWORK_ADDRESSES;
+
+export const REWARD_TOKEN_SPEED = "578703703703703";
+export const XVS = zksyncmainnet.XVS;
+export const STORE_XVS_REWARD_AMOUNT = parseUnits("6000", 18);
+export const RD_XVS_REWARD_AMOUNT = parseUnits("24000", 18);
```
Comment (chechu) [inline multisig/proposals/zksyncmainnet/vip-010/index.ts:11]:
```suggestion
export const RD_XVS_REWARD_AMOUNT = parseUnits("30000", 18);
```

File: vips/vip-368/bscmainnet.ts [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+export const COMPTROLLER = "0xfD36E2c2a6789Db23113685031d7F16329158384";
+export const ZKSYNCMAINNET_TREASURY = "0xB2e9174e23382f7744CebF7e0Be54cA001D95599";
+export const XVS = "0xcF6BB5389c92Bdda8a3747Ddb454cB7a64626C63";
+export const COMMUNITY_WALLET = "0xc444949e0054a23c44fc45789738bdf64aed2391";
+export const TREASURY = "0xF322942f644A996A617BD29c16bd7d231d9F35E9";
+export const ETH = "0x2170Ed0880ac9A755fd29B2688956BD959F933F8";
+
+export const RECEIVER_ADDRESS = ethers.utils.defaultAbiCoder.encode(["address"], [ZKSYNCMAINNET_TREASURY]);
+export const ADAPTER_PARAMS = ethers.utils.solidityPack(["uint16", "uint256"], [1, 300000]);
+export const XVS_AMOUNT = parseUnits("31500", 18);
```
Comment (chechu) [inline vips/vip-368/bscmainnet.ts:17]:
```suggestion
export const XVS_AMOUNT = parseUnits("37500", 18);
```

File: multisig/proposals/zksyncmainnet/vip-010/index.ts [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+      signature: "addRewardsDistributor(address)",
+      params: [REWARD_DISTRIBUTOR_CORE_0],
+    },
+    {
+      target: REWARD_DISTRIBUTOR_CORE_0,
+      signature: "setRewardTokenSpeeds(address[],uint256[],uint256[])",
+      params: [
+        [VWETH_CORE, VWBTC_CORE, VUSDT_CORE, VUSDC_E_CORE, VZK_CORE],
+        ["370370370370370", "370370370370370", "277777777777777", "555555555555555", "277777777777777"],
+        ["92592592592592", "92592592592592", "69444444444444", "138888888888888", "69444444444444"],
```
Comment (chechu) [inline multisig/proposals/zksyncmainnet/vip-010/index.ts:73]:
```suggestion
        ["740740740740740", "740740740740740", "555555555555554", "1111111111111110", "555555555555554"],
        ["185185185185184", "185185185185184", "138888888888888", "277777777777776", "138888888888888"],
```

File: multisig/proposals/zksyncmainnet/vip-010/index.ts (hunk 1 above)
Comment (GitGuru7) [reply multisig/proposals/zksyncmainnet/vip-010/index.ts]: [d5194f9](https://github.com/VenusProtocol/vips/pull/387/commits/d5194f9537c90cf0b2067354322ea48949f9f622)

File: vips/vip-368/bscmainnet.ts (hunk 3 above)
Comment (GitGuru7) [reply vips/vip-368/bscmainnet.ts]: [d5194f9](https://github.com/VenusProtocol/vips/pull/387/commits/d5194f9537c90cf0b2067354322ea48949f9f622)

File: multisig/proposals/zksyncmainnet/vip-010/index.ts (hunk 4 above)
Comment (GitGuru7) [reply multisig/proposals/zksyncmainnet/vip-010/index.ts]: [d5194f9](https://github.com/VenusProtocol/vips/pull/387/commits/d5194f9537c90cf0b2067354322ea48949f9f622)

File: multisig/simulations/zksyncmainnet/vip-010/index.ts [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+    before(async () => {
+      bridgeSigner = await initMainnetUser(BRIDGE, parseUnits("1", 18));
+
+      xvsVault = new ethers.Contract(XVS_VAULT_PROXY, XVS_VAULT_ABI, provider);
+      xvs = new ethers.Contract(XVS, XVS_ABI, provider);
+
+      await xvs.connect(bridgeSigner).mint(zksyncmainnet.VTREASURY, TREASURY_AMOUNT);
+    });
+
+    it("vTreasury should hold atleast 31500 XVS", async () => {
```
Comment (chechu) [inline multisig/simulations/zksyncmainnet/vip-010/index.ts:55]:
```suggestion
    it("vTreasury should hold atleast 37500 XVS", async () => {
```

**Review by chechu — APPROVED** [review]

**Pull request merged**